### ISSUE TITLE
implement Draft RFC Specification

### ISSUE THREAD

**Issue by mcuadros** [sevki/graphql#1]
The Draft was release this last Thursday in the React conference: http://facebook.github.io/graphql/
You have the intention of implement it?

**Comment by sevki** [comment]
Yeah I am working on it as we speak. this implementation was good enough for g2sd, but I decided to do some refactoring to make it more spec compliant, so other people can help me. should have something up this week.

**Comment by mcuadros** [comment]
Great.

Something similar to g2sd, was a suggestion from the Facebook team on the conference, maybe you can publish it on HN whit the updated lib. 

 Thx!

**Comment by sevki** [comment]
[g2sd](https://sevki.org/g2sd) is already on [github](https://github.com/sevki/g2sd) I'll wait a while for it to mature before I put it on HN or somewhere else. I don't think it's ready to hit the primetime just yet.
